students/EricAdams/Iterators_and_Generators_Packaging/generator_lab/generator_solution.py

```diff
@@ -0,0 +1,74 @@
+"""
+generator_solution.py
+
+Solutions for the generator lab excercises.
+
+"""
+
+
+def intsum():
+    '''sum using generator
+    '''
+    x = list(range(0, 1000))
```

Comment:
this kinda defeats the point of using a generator -- no need to create a full list ahead of time. you can just loop through the range object!